community-operators/iot-simulator/iot-simulator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,398 @@
+        "kind": "SimulatorProducer",
+        "metadata": {
+          "name": "producer1",
+          "namespace": "iot-simulator"
+        },
+        "spec": {
+          "simulator": "hono1",
+          "tenant": "myapp1.iot",
+          "type": "telemetry",
+          "protocol": "http",
+          "replicas": 1,
+          "numberOfDevices": 10
+        }
+      }
+      ]
+
+spec:
+  maturity: alpha
+  displayName: IoT simulator
+  description: |
+    A scalable IoT simulator, which simulates producers (MQTT or HTTP) and consumers.
+
+    ## Pre-requisites
+
+    You will need to have access to a Hono instance, including the messaging backend
+    and the HTTP endpoint of the device registry.
+
+    ## Creating an instance
+
+    Once you have the operator deployed, you need to set up at least one `Simulator` resource.
+    This will trigger the creation of the web console and deploy a Prometheus instance for
+    gathering metrics. The `Simulator` instance also contains the endpoint information for the
+    Hono instance.
+
+    ## Producers & Consumer
+
+    After a `Simulator` instance has been created, you can create `SimulatorProducer` and
+    `SimulatorConsumer` instances as necessary. You will need to reference the `Simulator`
+    instance.
+
+  version: 0.1.0
+  keywords: ['iot', 'messaging', 'amqp', 'mqtt', 'http']
+  labels:
+    app: iot-simulator
+  selector:
+    matchLabels:
+      app: iot-simulator
+
+  links:
+    - name: Documentation
+      url: https://dentrassi.de/iot-simulator
+    - name: Eclipse Hono™ Simulator
+      url: https://github.com/ctron/hono-simulator
+    - name: IoT Simulator Console
+      url: https://github.com/ctron/iot-simulator-console
+    - name: IoT Simulator Operator
+      url: https://github.com/ctron/iot-simulator-operator
+
+  maintainers:
+    - email: ctron@dentrassi.de
+      name: Jens Reimann
+
+  provider:
+    name: Jens Reimann
+
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: false
+
+  install:
+    strategy: deployment
+    spec:
+      deployments:
+        - name: iot-simulator-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                k8s-app: iot-simulator-operator
+            template:
+              metadata:
+                labels:
+                  k8s-app: iot-simulator-operator
+              spec:
+                serviceAccountName: iot-simulator-operator
+
+                containers:
+                  - name: operator
+                    image: 'docker.io/ctron/iot-simulator-operator:0.1'
+                    imagePullPolicy: Always
+
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
```

Comment:
This should be set to `metadata.annotations['olm.targetNamespaces']` to allow OLM to install the operator

Reply:
Ok, done.